databases/week2/homework.sql

```diff
@@ -0,0 +1,102 @@
+--Add a task with these attributes: title, description,
+-- created, updated, due_date, status_id, user_id
+ insert INTO task (title,description, created, 
+    updated, due_date, status_id, user_id);
+--Change the title of a task
+update mydb.task set title='do homework' where id=7;
+
+--Change a task due date
+update mydb.task set due_date='2024-07-10 20:16:15' where id =3;
+--Change a task status
+update mydb.task set status_id='1' where title='do homework';
+--Mark a task as complete
+UPDATE mydb.task set status_id='3' WHERE title='do homework';
```

Comment:
Try to not hardcode the status_id but look for the id where status = 'done'

Reply:
select title, name from task inner join status on 
task.status_id=status.id where name='Done';
is this your mean?

Reply:
dont need a join here, just a simple query on the status table to get the id of status 'done', and then put that query where you put in inside your update. `UPDATE task set status_id = (Select ...`